service-front/composer.json

```diff
@@ -53,7 +52,8 @@
     "autoload": {
         "psr-4": {
             "Application\\": "module/Application/src/",
-            "MakeLogger\\": "../shared/logging/module/MakeLogger/src/"
+            "MakeLogger\\": "../shared/logging/module/MakeLogger/src/",
+            "Alphagov\\Pay\\": "module/Alphagov/Pay/src/"
```

Comment:
should this be a shared component? i guess not?

Reply:
This used to be in its own git repo, and handed from OPG to Alphagov in case anyone else wanted to use it. In the event no-one did, and it got abandoned. It was pinned at old dependency versions, so blocked upgrade to PHP 8. So as a first step I've pulled it in , as is. In the long run it could be pulled inside the Make code base and not be a standalone component. But that would require tweaking namespaces which I'd ideally prefer to do in seperate PR rather than change too much stuff at once,  if you guys concur?